scala-generator/src/main/scala/models/generator/ScalaClientMethodConfig.scala

```diff
@@ -339,6 +339,7 @@ implicit def circeJsonDecoder[${asyncTypeParam(Some("Sync")).map(_+", ").getOrEl
     override def serverImports: String =
       s"""
          |import org.http4s.dsl.{io => _, _}
+         |import org.http4s.implicits._ 
```

Comment:
This should be conditional to http4s version (only 0.20)

Reply:
This is only in the `Http4s020` case class. Do I need to check for anything else?